ignek_intranet/modules/Count-Of/bnd.bnd

```diff
@@ -0,0 +1,4 @@
+Bundle-Name: Count-Of
+Bundle-SymbolicName: Count.Of
+Bundle-Version: 1.0.0
+Export-Package: Count.Of.constants
```

Comment:
In Bundle-Name, capital latter is not allowed. It should be like "ignek-counts-portlet".
Bundle-SymbolicName should be like "com.ignek.intranet.counts.portlet".
In package name, capital latter is not allowed. It should be like "com.ignek.intranet.counts.portlet".